### DISCUSSION TITLE
台灣團隊入圍蘋果設計大獎

### DISCUSSION THREAD

**Discussion by AngusGuei** [1101-GS3073-C/Python-Community-GS3073-C#408] [General]
恭喜《傾聽畫語》入圍蘋果設計大獎！
還是希望台灣遊戲產業能盡快跟上國際，有政府立案保護，多數人也能正確看待！

https://technews.tw/2022/06/10/behind-the-frame/

**Comment by lx3211** [comment]
畫畫與密室逃脫的結合真的滿新穎的

**Comment by NcuJerryKao** [comment]
https://apps.apple.com/tw/app/%E5%82%BE%E8%81%BD%E7%95%AB%E8%AA%9E/id1552126919
評價竟然有4.9顆星

**Comment by BingRong1217** [comment]
台灣在科技方面的發展也好厲害

**Comment by Ben6uo** [comment]
臺灣在科技上的發展真的好強

**Comment by limitx0** [comment]
傾聽畫語真的很好玩，故事也超感人的
推薦大家去玩

**Comment by Yilin20021016** [comment]
大推這個 是個很有心做的劇情
